pkgs/by-name/el/element-call/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+}:
+
+let
+  os = if stdenv.isDarwin then "osx" else "linux";
+  offlineCacheHash = {
+    linux = "sha256-xVpObAJEsHVS0+bvasQZ97PRRUdtvXpiV03qt29tz8c=";
+    osx = "sha256-bHc1FjvKDTL5apJqgXv7zk1z1WF68B3T7ibV7ZX3DWM=";
+  }.${os};
```

Comment:
Please use this style https://github.com/NixOS/nixpkgs/blob/1461dbbee35356d6aea102da072df380b0a5749c/pkgs/applications/editors/vscode/vscode.nix#L22-L38 to not fallback to linux on any other platform and throw correct.